Code_of_Conduct.md

```diff
@@ -0,0 +1,39 @@
+# Contributor Code of Conduct
```

Comment:
Why is this file here?

Reply:
Ok will remove

Reply:
sir this contributor code of conduct is not at all there from where you checked that

Reply:
@Sloth-Panda As your old PR is not merged yet, it shows it here. My mentors are inactive. Gets me into trouble! 😢

Reply:
@Sloth-Panda Why is this file still here?